doc/tutorials/beginner/develop_editor.md

```diff
@@ -0,0 +1,279 @@
+# Develop in an Editor
+
+In this section you will learn the basics of developing Panel apps in an editor:
+
+- serve your app with *autoreload* using `panel serve app.py --autoreload`
+- display the app in a *simple* browser tab inside your editor if possible
+- Inspect Panel objects via *hover* and `print`
+- Inspect a components parameters via `.param` and `.param._repr_html_()`
+
+:::{admonition} Note
+Some of the features demonstrated in this guide might require special configuration of your editor. For configuration we refer you to the [Resources](#resources) section below and general resources on the web.
+:::
+
+## Install the Dependencies
+
+Please make sure [Matplotlib](https://matplotlib.org/) and [Numpy](https://numpy.org/) are installed.
+
+::::{tab-set}
+
+:::{tab-item} conda
+:sync: conda
+
+``` bash
+conda install -y -c conda-forge matplotlib numpy
+```
+:::
+
+:::{tab-item} pip
+:sync: pip
+
+``` bash
+pip install matplotlib numpy
+```
+:::
+
+::::
+
+## Serve your app with autoreload
+
+A simple Panel app could look like the below.
+
+```python
+import panel as pn
+import numpy as np
+
+from matplotlib.figure import Figure
+
+ACCENT = "goldenrod"
+LOGO = "https://assets.holoviz.org/panel/tutorials/matplotlib-logo.png"
+
+pn.extension(sizing_mode="stretch_width")
+
+data = np.random.normal(1, 1, size=100)
+fig = Figure(figsize=(8,4))
+ax = fig.subplots()
+ax.hist(data, bins=20, color=ACCENT)
+
+component = pn.pane.Matplotlib(fig, format='svg', sizing_mode='scale_both')
+
+pn.template.FastListTemplate(
+    title="My App", sidebar=[LOGO], main=[component], accent=ACCENT
+).servable()
+```
+
+Copy the code above into a file named `app.py`.
+
+Save the file if you have not already done it.
+
+Serve the app by running the below command in a terminal.
+
+```bash
+panel serve app.py --autoreload
+```
+
+It should look like
+
+```bash
+$ panel serve app.py --autoreload
+2024-01-20 07:49:06,767 Starting Bokeh server version 3.3.3 (running on Tornado 6.4)
+2024-01-20 07:49:06,769 User authentication hooks NOT provided (default user enabled)
+2024-01-20 07:49:06,771 Bokeh app running at: http://localhost:5006/app
+2024-01-20 07:49:06,771 Starting Bokeh server with process id: 22100
+```
+
+Open [http://localhost:5006/app](http://localhost:5006/app) in a browser.
+
+It should look like
+
+![Panel served app](../../_static/images/develop_editor_panel_serve_before.png)
+
+Now change the
+
+- `ACCENT` value to `teal` and save the `app.py` file.
+- `bins` value to `15` and save
+- `title` value to `"My Matplotlib App"` and save
+
+It should look like
+
+<video controls="" poster="../../_static/images/develop_editor_panel_serve_after.png">
```

Comment:
I've both set video max-width: 100% inline and in .css file. But it does not seem  to work. I need help.

![image](https://github.com/holoviz/panel/assets/42288570/bf89c9a8-fbde-4423-a002-6431bda2767b)